OpenAPI/swagger.json

```diff
@@ -1158,327 +1234,367 @@
         }
       }
     },
-    "Flight" : {
-      "type" : "object",
-      "properties" : {
-        "id" : {
-          "type" : "integer",
-          "format" : "int64"
-        },
-        "pilot" : {
-          "type" : "object",
-          "properties" : {
-            "name" : {
-              "type" : "string"
+    "Flight": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "integer",
+          "format": "int64"
+        },
+        "pilot": {
+          "type": "object",
+          "properties": {
+            "name": {
+              "type": "string"
             },
-            "photoUrl" : {
-              "type" : "string"
+            "photoUrl": {
+              "type": "string"
             }
           }
         },
-        "clubName" : {
-          "type" : "string"
+        "clubName": {
+          "type": "string"
+        },
+        "teamName": {
+          "type": "string"
+        },
+        "aircraft": {
+          "$ref": "#/definitions/Aircraft"
+        },
+        "flightType": {
+          "type": "string",
+          "enum": ["JOJO", "Dreieck", "FAI Dreieck"]
         },
-        "teamName" : {
-          "type" : "string"
+        "flightStatus": {
+          "type": "string",
+          "enum": [
+            "Nicht in Wertung",
+            "In Wertung",
+            "Flugbuch",
+            "In Bearbeitung"
+          ]
         },
-        "aircraft" : {
-          "$ref" : "#/definitions/Aircraft"
+        "flightDistance": {
+          "type": "number"
         },
-        "flightType" : {
-          "type" : "string",
-          "enum" : [ "JOJO", "Dreieck", "FAI Dreieck" ]
+        "flightPoints": {
+          "type": "number"
         },
-        "flightStatus" : {
-          "type" : "string",
-          "enum" : [ "Nicht in Wertung", "In Wertung", "Flugbuch", "In Bearbeitung" ]
+        "fixes": {
+          "type": "array",
+          "items": {
+            "gpsAltitude": "number",
+            "latitude": "number",
+            "longitude": "number",
+            "timestamp": "number",
+            "elevation": "number"
+          }
+        },
+        "description": {
+          "type": "string"
+        },
+        "airbuddies": {
+          "type": "array",
+          "items": {
+            "flightId": "string",
+            "pilotName": "string"
+          }
         },
-        "flightDistance" : {
-          "type" : "number"
+        "takeoffTime": {
+          "type": "string"
         },
-        "flightPoints" : {
-          "type" : "number"
+        "landingTime": {
+          "type": "string"
```

Comment:
Add landing time to Flight model

Reply:
Flight model extended by "fixes" object with timestamps
Last timestamp == landingTime